scripts/deploy-backend.sh

```diff
@@ -57,6 +57,7 @@ set_cf_envs()
   "PROD_OIDC_OP_JWKS_ENDPOINT"
   "PROD_OIDC_OP_LOGOUT_ENDPOINT"
   "PROD_OIDC_OP_TOKEN_ENDPOINT"
+  "REDIS_URI"
```

Comment:
We have a conundrum here. In deployed environments, this needs to be `redis://localhost:6379` while for the CircleCI context it needs to be `redis://redis-server:6379`. Adding this env to CircleCI so that `deploy-backend.sh` can add it to whichever Cloud.gov app breaks `build-and-test` because redis isn't running locally on Docker but in a container. Another problem to resolve because we can't/don't use Docker within Cloud.gov.

Reply:
`gunicorn_start.sh` actually already differentiates docker vs deployed so perhaps this should be hard-set there instead of a finnicky CircleCI issue.

Reply:
If you used contexts the `build and test` could use a different context then the deploy workflow.

https://circleci.com/docs/contexts/

There would be a build context and a deploy context

Reply:
Resolved by adding this var directly via `gunicorn_start.sh`.